README.md

```diff
@@ -1,202 +1,147 @@
 # knot-cloud-sdk-js-authenticator
 
-KNoT Cloud authenticator service JavaScript library
+KNoT Cloud authenticator service JavaScript library.
 
-# Quickstart
+## Quickstart
 
-## Install
+### Install
 
 ```console
 npm install --save @cesarbr/knot-cloud-sdk-js-authenticator
 ```
 
-## Run
+### Run
 
 `KNoTCloudAuthenticator` connects to &lt;protocol&gt;://&lt;hostname&gt;:&lt;port&gt; using email and password as credentials. Replace this address with your authenticator instance and the credentials with valid ones.
 
 ```javascript
-const KNoTCloudAuthenticator = require('@cesarbr/knot-cloud-sdk-js-authenticator');
+const KNoTCloudAuthenticator = require("@cesarbr/knot-cloud-sdk-js-authenticator");
 
 const client = new KNoTCloudAuthenticator({
-  protocol: 'https',
-  hostname: 'auth.knot.cloud',
+  protocol: "https",
+  hostname: "auth.knot.cloud",
```

Comment:
Please, change to `api.knot.cloud` considering these operations are routed by the API Gateway.

Reply:
Done!